### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by Luffytaro22**
> # Calculator design 🔢 
> gif

> ## Features 🆕 
> * Calculator design based on the medium-fidelity `wireframe` 👍 
> * Use of `functional` react component 💯 
> * Created a `child` component ✔️

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <button type="button" className="operations right-operations">-</button>
+        <button type="button" className="operations">1</button>
+        <button type="button" className="operations">2</button>
+        <button type="button" className="operations">3</button>
+        <button type="button" className="operations right-operations">+</button>
+        <button type="button" className="operations" id="zero-button">0</button>
+        <button type="button" className="operations">.</button>
+        <button type="button" className="operations right-operations">=</button>
+      </div>
+    </div>
```
Comment (Shisui6) [inline src/components/Calculator.js]: As per the project requirements, the calculator component should have at least 1 child component. Please try to create an additional component and use it in your calculator component🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Luffytaro22, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- All CRA boilerplate removed ✅
- Calculator component added ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
+import React from 'react';
+
+// Child component
+function CalculatorButton({ className, text, id }) {
+  return (
+    <button type="button" className={className} id={id}>
+      {text}
+    </button>
+  );
+}
```
Comment (Ibizugbe) [inline src/components/Calculator.js:11]: [OPTIONAL] Nice job splitting your codebase into sub-components 😉. Kindly remember that the reason this is done is for the purpose of separation of concerns and reusability😉 which makes your code maintainable and reasuable😉. Hence, I will recommend you take this component to a separate file as this is not just only professional but also strongly fulfills the purpose of separation of concerns making your codebase easily usable and maintainable👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Luffytaro22 👋👋, 

StandingOvationGIF (9)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Linters are passing👍.
- [x] A child component has been created as requested by the previous reviewer👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**